deposit/Move.toml

```
@@ -0,0 +1,9 @@
+[package]
+name = "deposit"
+version = "0.0.0"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/move-language/move.git", subdir = "language/move-stdlib", rev = "main" }
```

I wonder if it would be better to use our repository, just in case if we change something there.

```suggestion
MoveStdlib = { git = "https://github.com/eigerco/substrate-move.git", subdir = "language/move-stdlib", rev = "main" }
```